dataflux_pytorch/tests/test_dataflux_lightning_gcs_filesystem.py

```diff
@@ -16,7 +17,9 @@ def setUp(self):
         self.bucket = fake_gcs.Bucket("fake_bucket")
         self.client = fake_gcs.Client()
         self.fake_gcs = GCSFileSystem(project_name=self.project_name,
+                                      debug=False,
                                       storage_client=self.client)
+        dist.init_process_group("gloo")
```

Comment:
Seems like testing doesn't like this being spun up in the fake implementation. I would either gate the printouts behind the debug flag (and pass false for testing) or I would mock `dist.get_rank()` out of all tests where it causes a failure.

Reply:
Removed